### PR TITLE
Add ShellCheck support

### PR DESCRIPTION
**Pull request by turino**

### CODE REVIEW

**Comment by tysongach** [comment]
[Add ShellCheck for shell files?](https://trello.com/c/hwPBryoA/478-add-shellcheck-for-shell-files)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ To test Stripe payments on staging use this fake credit card number.
 The main Hound app (this app) receives PR hooks from GitHub, then it
 communicates with the [Linters](https://github.com/houndci/linters) app
 (or a few individual linter services) to review changed files in the PR.
-Linters communcate back with vioaltions they found, and the Hound app sends
+Linters communicate back with violations they found, and the Hound app sends
```
Comment (salbertson) [inline CONTRIBUTING.md]: 👀

File: app/views/pages/configuration.haml [hunk 2]
```diff
@@ -563,7 +565,7 @@
         %em.code-inline .sass-lint.yml
         file to your project,
         specify your desired configuration
-        and refreence the file in your
+        and reference the file in your
```
Comment (salbertson) [inline app/views/pages/configuration.haml]: Awesome, thanks for catching and changing these spelling errors.

**Comment by salbertson** [comment]
@turino is everything set up for this to work with [linters](https://github.com/houndci/linters)?

**Comment by turino** [comment]
@salbertson yes, linter support has been added/merged: https://github.com/houndci/linters/pull/155/

**Pull request merged**